src/sqlfluff/rules/L028.py

```diff
@@ -24,7 +24,7 @@ class Rule_L028(Rule_L025):
         FROM foo
 
     | **Best practice**
-    |  Remove all the reference or reference all the fields.
+    |  Remove or add references to all fields.
```

Comment:
```suggestion
    |  Add or remove references to all fields.
```
NIT: reads weird the other way round

Reply:
Updated.